homeassistant/components/nextcloud/sensor.py

```diff
@@ -186,108 +198,110 @@
         suggested_display_precision=1,
         suggested_unit_of_measurement=UnitOfInformation.MEGABYTES,
     ),
-    SensorEntityDescription(
+    NextcloudSensorEntityDescription(
         key="jit_kind",
         translation_key="nextcloud_jit_kind",
         entity_category=EntityCategory.DIAGNOSTIC,
         entity_registry_enabled_default=False,
     ),
-    SensorEntityDescription(
+    NextcloudSensorEntityDescription(
         key="jit_opt_flags",
         translation_key="nextcloud_jit_opt_flags",
         entity_category=EntityCategory.DIAGNOSTIC,
         entity_registry_enabled_default=False,
     ),
-    SensorEntityDescription(
+    NextcloudSensorEntityDescription(
         key="jit_opt_level",
         translation_key="nextcloud_jit_opt_level",
         entity_category=EntityCategory.DIAGNOSTIC,
         entity_registry_enabled_default=False,
     ),
-    SensorEntityDescription(
+    NextcloudSensorEntityDescription(
         key="opcache_statistics_blacklist_miss_ratio",
         translation_key="nextcloud_opcache_statistics_blacklist_miss_ratio",
         entity_category=EntityCategory.DIAGNOSTIC,
         entity_registry_enabled_default=False,
         native_unit_of_measurement=PERCENTAGE,
     ),
-    SensorEntityDescription(
+    NextcloudSensorEntityDescription(
         key="opcache_statistics_blacklist_misses",
         translation_key="nextcloud_opcache_statistics_blacklist_misses",
         entity_category=EntityCategory.DIAGNOSTIC,
         entity_registry_enabled_default=False,
     ),
-    SensorEntityDescription(
+    NextcloudSensorEntityDescription(
         key="opcache_statistics_hash_restarts",
         translation_key="nextcloud_opcache_statistics_hash_restarts",
         entity_category=EntityCategory.DIAGNOSTIC,
         entity_registry_enabled_default=False,
     ),
-    SensorEntityDescription(
+    NextcloudSensorEntityDescription(
         key="opcache_statistics_hits",
         translation_key="nextcloud_opcache_statistics_hits",
         entity_category=EntityCategory.DIAGNOSTIC,
         entity_registry_enabled_default=False,
     ),
-    SensorEntityDescription(
+    NextcloudSensorEntityDescription(
         key="opcache_statistics_last_restart_time",
         translation_key="nextcloud_opcache_statistics_last_restart_time",
         device_class=SensorDeviceClass.TIMESTAMP,
         entity_category=EntityCategory.DIAGNOSTIC,
         entity_registry_enabled_default=False,
+        value_fn=lambda val: datetime.fromtimestamp(cast(int, val), tz=UTC),
```

Comment:
We have this util:

https://github.com/home-assistant/core/blob/f1ec99b9c9f67e87387203e178c945d69322b5c9/homeassistant/util/dt.py#L151

Reply:
@ mib1185 Could you let me know if you plan to implement the suggested change?